Source/JavaScriptCore/llint/WebAssembly.asm

```diff
@@ -298,25 +298,13 @@ macro restoreCalleeSavesUsedByWasm()
 end
 
 macro preserveGPRsUsedByTailCall(gpr0, gpr1)
-    if ARM64 or ARM64E
-        storepairq gpr0, gpr1, CodeBlock[sp]
-    elsif ARMv7 or X86_64 or X86_64_WIN or RISCV64
-        storep gpr0, CodeBlock[sp]
-        storep gpr1, Callee[sp]
-    else
-        error
-    end
+    storep gpr0, CodeBlock[sp]
+    storep gpr1, ArgumentCountIncludingThis[sp]
```

Comment:
What is the reason of this change?

Reply:
I changed the slot that got saved to (callee -> ACIT), so it can't be an stp anymore. That is because callee needs to be set by the caller now